src/pattern-library/components/patterns/navigation/LinkButtonPage.js

```diff
@@ -0,0 +1,197 @@
+import Next from '../../LibraryNext';
+
+export default function LinkButtonPage() {
+  return (
+    <Library.Page
+      title="LinkButton"
+      intro={
+        <p>
+          <code>LinkButton</code> is a presentational component that styles a
+          button to appear as a link.
+        </p>
+      }
+    >
+      <Library.Pattern title="Status">
+        <p>
+          <code>LinkButton</code> is a re-implementation of a legacy component
+          with the same name.
+        </p>
+
+        <Library.Example title="Migrating to this component">
+          <p>
+            <code>LinkButton</code> styling approach has changed. Instead of
+            defining its own inconsistent set of <code>variant</code>s, it
+            applies the same design patterns as <code>Link</code>. Most styling
+            control is handled by the <code>color</code> and{' '}
+            <code>underline</code> props.
+          </p>
+          <Next.Changelog>
+            <Next.ChangelogItem status="breaking">
+              Prop name:{' '}
+              <s>
+                <code>buttonRef</code>
+              </s>{' '}
+              ➜ <code>elementRef</code>
+            </Next.ChangelogItem>
+            <Next.ChangelogItem status="breaking">
+              Most <code>variant</code> values have been removed
+            </Next.ChangelogItem>
+            <Next.ChangelogItem status="breaking">
+              Prop name:{' '}
+              <s>
+                <code>className</code>
+              </s>{' '}
+              ➜ No longer supported
+            </Next.ChangelogItem>
+          </Next.Changelog>
+        </Library.Example>
+      </Library.Pattern>
+
+      <Library.Pattern title="Usage">
+        <Next.Usage componentName="LinkButton" />
+        <Library.Example>
+          <Library.Demo title="Basic LinkButton" withSource>
+            <LinkButton>Example of LinkButton</LinkButton>
+          </Library.Demo>
+        </Library.Example>
+      </Library.Pattern>
+
+      <Library.Pattern title="Props">
+        <p>
+          <code>color</code> applies to link text and hovered link text.
+        </p>
+        <Library.Example title="color">
+          <Library.Demo title="color: 'brand' (default)" withSource>
+            <LinkButton color="brand" underline="hover">
+              About this version
+            </LinkButton>
+          </Library.Demo>
+
+          <Library.Demo title="color: 'text-light'" withSource>
+            <LinkButton color="text-light" underline="hover">
+              Show replies (7)
+            </LinkButton>
+          </Library.Demo>
+          <Library.Demo title="color: 'text'" withSource>
+            <p className="text-color-text">
+              <LinkButton color="text">Page notes</LinkButton>{' '}
+            </p>
+          </Library.Demo>
+        </Library.Example>
+
+        <Library.Example title="underline">
+          <Library.Demo title="underline:'none' (default)" withSource>
```

Comment:
I suggest that, as with `Link`, we emphasize the importance of underlines when the link has surrounding text or appears in some other context where its clickability is non-obvious.